pkg/util/util.go

```diff
@@ -20,6 +20,15 @@ import (
 	"io"
 )
 
+// Encode writes data of specified interfaces to io writer
```

Comment:
suggestion: Encode writes the multiple inputs of different possible datatypes to the io writer.